parachain/pallets/basic-channel/src/outbound/mod.rs

```diff
@@ -30,14 +30,22 @@ pub use weights::WeightInfo;
 )]
 #[scale_info(skip_type_params(M, N))]
 #[codec(mel_bound())]
-pub struct MessageBundle<M: Get<u32>, N: Get<u32>> {
+pub struct MessageBundle<M: Get<u32>, N: Get<u32>, AccountId> {
 	source_channel_id: u8,
+	account: AccountId,
 	/// Unique nonce for to prevent replaying bundles
 	#[codec(compact)]
 	nonce: u64,
-	messages: BoundedVec<Message<M>, N>,
+	messages: BoundedVec<EnqueuedMessage<AccountId, M>, N>,
 }
 
+#[derive(
+	Encode, Decode, CloneNoBound, PartialEqNoBound, RuntimeDebugNoBound, MaxEncodedLen, TypeInfo,
+)]
+#[scale_info(skip_type_params(AccountId, M))]
+#[codec(mel_bound())]
```

Comment:
I think this may be needed to ensure that `AccountId` is substituted with something that implements `MaxEncodedLen`.
```suggestion
#[codec(mel_bound(AccountId: MaxEncodedLen))]
```